File: VB/Q506373/SimpleCommand.vb
Imports System
Imports System.Linq
Imports System.Windows.Input

Namespace Q506373
	Public Class SimpleCommand
		Implements ICommand

		Public Function CanExecute(ByVal parameter As Object) As Boolean Implements ICommand.CanExecute
			Return True
		End Function
		Public Event CanExecuteChanged As EventHandler Implements ICommand.CanExecuteChanged
		Public Sub Execute(ByVal parameter As Object) Implements ICommand.Execute
			Console.WriteLine("SimpleCommand: Action!")
		End Sub
	End Class
End Namespace

File: CS/Q506373/SimpleCommand.cs
using System;
using System.Linq;
using System.Windows.Input;

namespace Q506373 {
    public class SimpleCommand : ICommand {
        public bool CanExecute(object parameter) {
            return true;
        }
        public event EventHandler CanExecuteChanged;
        public void Execute(object parameter) {
            Console.WriteLine("SimpleCommand: Action!");
        }
    }
}
